src/FlexView.js

```diff
@@ -7,8 +7,10 @@ import omit from 'lodash/omit';
 import some from 'lodash/some';
 import { t, props } from 'tcomb-react';
 
+const IS_DEV = process.env.NODE_ENV !== 'production';
```

Comment:
I think we can be smarter in the future, but for now I'd leave the check inline (I mean the entire `process.env.NODE_ENV`) so that our builds can strip the useless code below. With a variable like this, it doesn't currently work in our average project build